geps/gep-3155/index.md

```diff
@@ -0,0 +1,205 @@
+# GEP-3155: Complete Backend mTLS Configuration
```

Comment:
I know and _most_ people reading this also know that "mTLS" here means "mutual TLS between parties", _without_ the additional Service Mesh concepts that make "mTLS" mean "mutual TLS between services with automatically-managed, rapidly rotated certificates".

But some people, when they read "mTLS", read the latter, not the former.

Could we please change this to "mutual TLS" throughout the document, and put a caveat at the top that "mutual TLS" in this document is used solely to refer to the two-way handshake between Gateway and backend, and _no other mutual TLS use cases_?

@mkosieradzki, I know this may seem pedantic and stupid, but in almost every conversation we've had about client certificate usage, _someone_ has been confused by the use of "mTLS". I just want to avoid having the same issue here as well.

Reply:
Thank you, I was not aware of that. I will update the doc accordingly to avoid confusion with managed TLS or managed mutual TLS.